src/styles/blocks/_card.scss

```diff
@@ -0,0 +1,106 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  text-align: left;
+  justify-content: center;
+
+  padding: 32px 16px 16px;
+
+  width: $card-width;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+
+    color: #060b35;
+  }
+
+  &__productCode {
```

Comment:
Be consistent about classes naming, in some cases you used camelCase, in some cases `-` as separator, let's use one style for all project (usually we are using `-` to separate words in BEM classes)